src/query/interpreter.cpp

```diff
@@ -3954,6 +3954,55 @@ PreparedQuery PrepareDatabaseInfoQuery(ParsedQuery parsed_query, bool in_explici
         return std::pair{results, QueryHandlerResult::COMMIT};
       };
 
+      break;
+    }
+    case DatabaseInfoQuery::InfoType::METRICS: {
+#ifdef MG_ENTERPRISE
+      if (!memgraph::license::global_license_checker.IsEnterpriseValidFast()) {
+        throw QueryRuntimeException("SHOW METRICS INFO command is only available with a valid Enterprise License!");
+      }
+#else
+      throw QueryRuntimeException("SHOW METRICS INFO command is only available in Memgraph Enterprise build!");
+#endif
+      header = {"name", "type", "metric type", "value"};
+      handler = [storage = current_db.db_acc_->get()->storage()] {
+        auto info = storage->GetBaseInfo();
+        auto metrics_info = storage->GetMetrics();
+        std::vector<std::vector<TypedValue>> results;
```

Comment:
All this code is good, to make it better:
- info and metrics_info can be const
- use std::ranges::sort